api/src/main/java/io/minio/BucketArgs.java

```diff
@@ -60,15 +60,28 @@ protected void validateName(String name) {
       }
     }
 
-    @SuppressWarnings("unchecked") // Its safe to type cast to T as T is inherited by this class
+    private static void validateRegion(String region) {
+      if (region != null && region.isEmpty()) {
```

Comment:
This condition should be:

`if (region == null || region.isEmpty())`

Reply:
Actually `null` region is allowed. When region is `null` we need to probe or use default region. However if region is provided, it should not be empty.